### DISCUSSION TITLE
Google Stadia Input

### DISCUSSION THREAD

**Discussion by JakeIsHench** [AntiMicroX/antimicrox#823] [Ideas]
Antimicrox has been great so far when using a Google Stadia controller, however the "google assistant" button isn't shown as an input, so if possible some way to utilise it would be helpful with more button heavy games like Euro Truck Sim

**Comment by pktiuk** [comment]
Button visibility and usability depends on support provided by an underlying SDL library. https://github.com/libsdl-org/SDL
